hack/chains/config.sh

```diff
@@ -78,29 +78,54 @@ case "$1" in
 
     ;;
 
+  rekor-local )
+    REKOR_SERVER=$(kubectl get ingress -n rekor-server -o yaml|yq e '.items[.spec].spec.rules[.host].host')
+    $0 "transparency.url: \"$REKOR_SERVER\"" $2
+
+    ;;
+
   * )
     # Avoid clearing all config if no param is given
     [[ -z $1 ]] && $0 get && exit
 
+
     # Use yq to convert the input to a single line of json
     # so we can use yaml or json for the input
     PATCH=$( echo "$1" | yq -o=json --indent=0 e - )
 
     if [[ $2 == "--dry-run" ]]; then
-      # Just show the desired config
-      echo "$PATCH" | yq e -P 'sort_keys(..)' -
+      if [[ $1 == "rekor-default" ]]; then
+        echo $($0 get | yq -o=json --indent=0 e 'del(."transparency.url")' - )
```

Comment:
I get the dry run output is a bit different for removing a key. But we output yaml for all the others, so it seems inconsistent to use one-liner json for this one.

(Another problem that would go away if we just make an assumption about the default value of transparency.url.)